SCClassLibrary/Common/Control/Server.sc

```diff
@@ -1,10 +1,12 @@
 ServerOptions {
-	// order of variables is important here. Only add new instance variables to the end.
+
+	// order of variables is important here. You may add new instance variables after outDevice.
```

Comment:
As I noted below, this is only my interpretation at a quick glance at one code file. I'd really prefer it if this was verified more thoroughly so we don't commit misinformation.